From ce99fcf8441a97ef58555ead837498eeca4aaff3 Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@apache.org>
Date: Mon, 16 Jul 2012 17:30:50 +0000
Subject: [PATCH 1279/1357] MR1: HADOOP-8552. Conflict: Same security.log.file for multiple users. (kkambatl via tucu)

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/branches/branch-1@1362154 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit c1f59496ccba92261d85cc274450e5c759f0129d)

Reason: log4j error printed out in task process
Ref:
Author: Sandy Ryza
---
 hadoop-mapreduce1-project/conf/log4j.properties |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hadoop-mapreduce1-project/conf/log4j.properties b/hadoop-mapreduce1-project/conf/log4j.properties
index c9044d3..ae3fcc2 100644
--- a/hadoop-mapreduce1-project/conf/log4j.properties
+++ b/hadoop-mapreduce1-project/conf/log4j.properties
@@ -61,7 +61,7 @@ log4j.appender.TLA.layout.ConversionPattern=%d{ISO8601} %p %c: %m%n
 #
 #Security audit appender
 #
-hadoop.security.log.file=SecurityAuth.audit
+hadoop.security.log.file=SecurityAuth-${user.name}.audit
 log4j.appender.DRFAS=org.apache.log4j.DailyRollingFileAppender 
 log4j.appender.DRFAS.File=${hadoop.log.dir}/${hadoop.security.log.file}
 
-- 
1.7.0.4

